pkg/config/configuration.go

```diff
@@ -279,6 +280,11 @@ type ComponentsSpec struct {
 	Deny []string `json:"deny,omitempty" yaml:"deny,omitempty"`
 }
 
+// WasmSpec describes the configuration for Wasm across all Dapr components.
+type WasmSpec struct {
+	StrictSandbox *bool `json:"strictSandbox,omitempty" yaml:"strictSandbox,omitempty"`
```

Comment:
If the default value is "false" (i.e. strict sandbox not enabled), consider making this non-nullable, i.e.:


```suggestion
	StrictSandbox bool `json:"strictSandbox,omitempty" yaml:"strictSandbox,omitempty"`
```

Reply:
Done. Make it non-nullable.